### PR TITLE
Move the PreviewAuthenticationListener to the core-bundle

### PR DESCRIPTION
**Pull request by bytehead**
If you use Contao just as a bundle (unmanaged edition, see https://docs.contao.org/dev/getting-started/initial-setup/symfony-application/), you won't get redirected to the login screen if you access the preview url unauthenticated.

As the preview is a feature of the core-bundle, this PR moves the according authentication listener from the manager-bundle to the core-bundle.

### CODE REVIEW

File: core-bundle/src/Resources/config/listener.yml [hunk 1]
```diff
@@ -140,6 +140,17 @@ services:
             # The priority must be higher than the one of the Twig exception listener (defaults to -128)
             - { name: kernel.event_listener, event: kernel.exception, method: onKernelException, priority: -96 }
 
+    contao.listener.preview_authentication:
+        class: Contao\ManagerBundle\EventListener\PreviewAuthenticationListener
```
Comment (leofeyer) [inline core-bundle/src/Resources/config/listener.yml]: The class name still references `ManagerBundle`. Is this correct?

**Review by leofeyer — CHANGES_REQUESTED** [review]
I wonder why the `PreviewAuthenticationListenerTest` class is not listed as moved file. Did you forget to delete the file in the manager bundle?

File: core-bundle/src/Resources/config/listener.yml (hunk 1 above)
Comment (bytehead) [reply core-bundle/src/Resources/config/listener.yml]: Copy pasta 🙈 Fixed in 2e4011d15.

**Comment by bytehead** [comment]
> Did you forget to delete the file in the manager bundle?

There were no tests before...

**Review by aschempp — APPROVED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**